File: code/connectTest.py
import mysql.connector

try:
    cnx = mysql.connector.connect(user='testuser', 
                                  password='atytestpw',
                                  host='localhost', 
                                  database='testdb')
    cursor = cnx.cursor()
    print("db connected")

    # query = ("CREATE TABLE test (" +
    #          "eno CHAR(5), ename VARCHAR(30) NOT NULL, PRIMARY KEY (eno))")
    # cursor.execute(query)
    # print('Table test created')

    query = ("INSERT INTO test(eno, ename) VALUES (\"abcde\", \"Mr.Random\")")
    cursor.execute(query)
    cnx.commit()    # Store the insert to database permanently 
    print('Inserted')

    query = ("SELECT eno, ename FROM test")
    cursor.execute(query)
    for (eno, ename) in cursor:
        print(eno)
        print(ename)
    print("Select executed")

    # query = ("DROP TABLE test")
    # cursor.execute(query)
    # print("Table test dropped")
    
    cursor.close()
except mysql.connector.Error as err:
    print(err)
finally:
    cnx.close()

File: code/php/connectTest.php
<!DOCTYPE html>
<html lang="en">
<head>
</head>

<body>


<!-- Connecting to database here. -->
<?php
try{
    echo "hello1 <br>";   
    // $host = "cosc360.ok.ubc.ca";
    // $database = "db_11888757";
    // $user = "11888757";
    // $password = "11888757";
    // $connString = "mysql:host=cosc360.ok.ubc.ca;dbname=db_11888757";


    $host = "localhost";
    $database = "testdb";
    $user = "testuser";
    $password = "atytestpw";
    $connString = "mysql:host=localhost;dbname=testdb";

    $pdo = new PDO($connString, $user, $password);
    // $conn = mysqli_connect($host, $user, $password, $database);

    echo "hello2"."<br>";
    echo "<br>";
    
    $sql = "SELECT * FROM test";
    echo $sql."<br>";
    
    // $result = mysqli_query($conn, $sql);
    // echo "Result fetched <br> ";
    // echo $result;

    $result = $pdo->query($sql);
    while ($row = $result->fetch()) {
        echo $row['eno']."-".$row['ename']."<br>";
    }

    echo "<br>";

    // method 1
    $sql = "SELECT * FROM test WHERE eno = ?";
    $stmt = $pdo->prepare($sql);
    $stmt->bindValue(1, "E0001");

    // method 2
    // $sql = "SELECT * FROM test WHERE eno = :eno";
    // $stmt = $pdo->prepare($sql);
    // $stmt->bindValue('eno', "E0001");

    echo $sql."<br>";
    // echo $stmt."<br>"; THIS DOES NOT WORK
    
    $stmt->execute();
    while ($row = $stmt->fetch(PDO::FETCH_ASSOC)) {
        echo $row['eno']."-".$row['ename']."<br>";
    }


}catch(Exception $e){
    echo print($e);
    echo mysqli_connect_error();
}


?>

</body>
</html>

